libflux/src/core/semantic/nodes.rs

```diff
@@ -88,6 +90,7 @@ pub enum Statement {
     Return(ReturnStmt),
     Test(Box<TestStmt>),
     Builtin(BuiltinStmt),
+    Invalid(Box<BadStmt>),
```

Comment:
For consistency can you name the enum `Bad`? I get that its a bit annoying but the consistency with the rest of the variants and naming convention is important.

Reply:
Sure. Maybe we can consider changing the enum in the AST too in the future.